### PR TITLE
Avoid confusion of inplace LM masking

### PR DESCRIPTION
**Pull request by ghost**
Fix confusion of LM masking that happens inplace. Discussed in https://github.com/huggingface/pytorch-pretrained-BERT/issues/243 and https://github.com/huggingface/pytorch-pretrained-BERT/issues/226

### CODE REVIEW

**Review by rodgzilla — APPROVED** [review]

**Comment by thomwolf** [comment]
Great, thanks @tholor!
